cli/cmd/util.go

```diff
@@ -34,8 +34,14 @@ func verifyCoordinator(host string, configFilename string, insecure bool) ([]*pe
 	}
 
 	// get latest config from github if none specified
-	fmt.Println("No era config file specified, getting latest config from github.com/edgelesssys/marblerun/releases/latest/download/coordinator-era.json")
-	resp, err := http.Get("https://github.com/edgelesssys/marblerun/releases/latest/download/coordinator-era.json")
+	coordinatorVersion, err := getCoordinatorVersion()
+	if err != nil {
+		return nil, err
+	}
+	eraURL := fmt.Sprintf("https://github.com/edgelesssys/marblerun/releases/%s/download/coordinator-era.json", coordinatorVersion)
```

Comment:
```suggestion
	eraURL := fmt.Sprintf("https://github.com/edgelesssys/marblerun/releases/download/%s/coordinator-era.json", coordinatorVersion)
```
Seems that the URL for a specific version is https://github.com/edgelesssys/marblerun/releases/download/v0.3.0/coordinator-era.json
While for latest there is a shortcut:
https://github.com/edgelesssys/marblerun/releases/latest/download/coordinator-era.json